snmp/datadog_checks/snmp/data/profiles/dell-poweredge.yaml

```diff
@@ -16,10 +15,45 @@ extends:
 device:
   vendor: "dell"
 
-
-sysobjectid: 1.3.6.1.4.1.674.10892.1.*
+sysobjectid:
+  - 1.3.6.1.4.1.674.10892.1 # Poweredge R640
+  - 1.3.6.1.4.1.674.10892.1.*
```

Comment:
From what I could see on the DELL forums/docs, sys Object ID may all be  1.3.6.1.4.1.674.10892.1 for poweredges.
But let's keep it that way.